packages/builder/src/builtin-tasks/run.ts

```diff
@@ -35,23 +56,7 @@ export default function () : void {
           //await run(TASK_COMPILE);
         }
 
-        log(`Running script ${script} in a subprocess so we can wait for it to complete`);
-
-        try {
-          process.exitCode = await runScriptWithAlgob(
-            runtimeArgs,
-            script
-          );
-        } catch (error) {
-          throw new BuilderError(
-            ERRORS.BUILTIN_TASKS.RUN_SCRIPT_ERROR,
-            {
-              script,
-              error: error.message,
-            },
-            error
-          );
-        }
+        await runSingleScript(runtimeArgs, script, log)
```

Comment:
same here, `noCompile` can be removed.
Also, it should be able to run multiple scripts (please see the spec).

Reply:
This is `run`.

Reply:
`noCompile` removed. `run` currently doesn't support multiple scripts.

Reply:
Running multiple scripts sounds already simple to add. If you don't add it here then please add a new task (thought if it's easy let's do it here)